app2/src/main/java/com/example/recipebox/ui/theme/Color.kt

```diff
@@ -2,7 +2,7 @@ package com.example.recipebox.ui.theme
 
 import androidx.compose.ui.graphics.Color
 
-val Purple200 = Color(0xFFBB86FC)
```

Comment:
I would recommend changing the colors the ones in your design document, rather than using the default IntelliJ generated colors class. I tried to color sample from your design doc and match it closely